cli/src/commands/rebase.rs

```diff
@@ -392,9 +392,10 @@ fn rebase_descendants_transaction(
     rebase_options: RebaseOptions,
 ) -> Result<(), CommandError> {
     workspace_command.check_rewritable(old_commits.iter().ids())?;
-    let (skipped_commits, old_commits) = old_commits
-        .iter()
-        .partition::<Vec<_>, _>(|commit| commit.parents() == new_parents);
+    let (skipped_commits, old_commits) = old_commits.iter().partition::<Vec<_>, _>(|commit| {
+        let parents: Vec<_> = commit.parents().try_collect().unwrap();
+        parents == new_parents
```

Comment:
nit: this could be rewritten to `commit.parent_ids().iter().eq(new_parents.iter().ids())`, but might be better to do that separately.

Reply:
Good idea. Done in a separate commit at the base of this PR.